highlight.io/pages/docs/[[...doc]].tsx

```diff
@@ -1033,7 +1033,10 @@ export default function DocPage({
 												EnterpriseSelfHostCalendlyComponent,
 												DocsCardGroup,
 												h1: (props) => (
-													<h4 {...props} />
+													<h1
+														{...props}
+														className="docH1"
+													/>
 												),
```

Comment:
looks like this might be working, but the main titles on pages are still h3 tags whereas we want them to be h1s

![Screenshot from 2024-10-04 17-52-53](https://github.com/user-attachments/assets/ac8fccc6-9f00-4632-a66a-a776e6a3ab6e)